packages/cloud-cognitive/src/global/js/utils/motionConstants.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { getBezierValues } from './getBezierValues';
+
+export const DURATIONS = {
+  // Micro-interactions such as button and toggle
+  fast01: 0.07, // '70ms'
+  // Micro-interactions such as fade
+  fast02: 0.11, // '110ms'
+  // Micro-interactions, small expansion, short distance movements
+  moderate01: 0.15, // '150ms'
+  // Expansion, system communication, toast
+  moderate02: 0.24, // '240ms'
+  // Large expansion, important system notifications
+  slow01: 0.4, // '400ms'
+  //  Background dimming
+  slow02: 0.7, // '700ms'
+};
```

Comment:
I think in this file we should use the `@carbon/motion` package since they have durations already, this way we are receiving future updates.